doc/api/esm.md

```diff
@@ -1306,19 +1313,20 @@ export async function transformSource(source,
     };
   }
   // Defer to Node.js for all other sources.
-  return defaultTransformSource(
-    source, context, defaultTransformSource);
+  return null;
 }
 ```
 
-#### <code>getGlobalPreloadCode</code> hook
+#### <code>globalPreloadCode</code>
```

Comment:
Any reason to have every other hook start with a verb and be a function but make this one a constant?